modules/main/src/main/env/docker/wasabi/Dockerfile

```diff
@@ -13,34 +13,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ###############################################################################
-from centos
+FROM openjdk:8-jre-alpine
 
-ENV WASABI_MODULE wasabi-main
-ENV WASABI_UI_MODULE wasabi-ui-main
-ENV WASABI_PORT 8080
-ENV WASABI_JMX_PORT 8090
-ENV WASABI_DEBUG_PORT 8180
+COPY ./ /usr/local/${application.name}
```

Comment:
Let's add a todo on making these magic files that appeared in docker more standard by adding from explicit path